### PR TITLE
Added support for opening files with Turntable

### PR DESCRIPTION
**Pull request by weruder**

### CODE REVIEW

File: Clients/Win32 Player/Code/Game/Main_Win32.cpp [hunk 1]
```diff
@@ -108,6 +108,32 @@ void HandleFileDrop(WPARAM wParam)
...
+
+    argList = CommandLineToArgvW(GetCommandLineW(), &numArgs);
+
+    if (numArgs > 1 && argList != NULL)
+    {
+        for (int i = 0; i < numArgs - 1; ++i)
+        {
+            if (!SongManager::instance->IsPlaying())
+            {
+                Console::instance->RunCommand(WStringf(L"play \"%s\"", argList[i+1]), true);
```
Comment (picoriley) [inline Clients/Win32 Player/Code/Game/Main_Win32.cpp]: Mention that the first argument is the app's path somewhere in this function c:

File: Clients/Win32 Player/Code/Game/Main_Win32.cpp [hunk 2]
```diff
@@ -414,9 +441,24 @@ void Shutdown()
...
+	std::wstring currentPath = std::wstring(buffer) + L"\\Turntable.exe";
+
+	if (!FileExists(currentPath))
+	{
+		int numArgs;
+		LPWSTR* argList;
+		argList = CommandLineToArgvW(GetCommandLineW(), &numArgs);
+		std::wstring path = GetFileDirectory(argList[0]);
+		SetCurrentDirectory(path.c_str());
+        LocalFree(argList);
```
Comment (picoriley) [inline Clients/Win32 Player/Code/Game/Main_Win32.cpp]: Is this formatted correctly offline?

Comment (weruder) [reply Clients/Win32 Player/Code/Game/Main_Win32.cpp]: Yes, it's formatted correctly locally.

File: Clients/Win32 Player/Code/Game/Main_Win32.cpp (hunk 1 above)
Comment (weruder) [reply Clients/Win32 Player/Code/Game/Main_Win32.cpp]: Added!

**Pull request merged**